deps/build.jl

```diff
@@ -7,6 +7,13 @@ builddir = joinpath(BinDeps.depsdir(mpi), "build")
 prefix = joinpath(BinDeps.depsdir(mpi), "usr")
 src = joinpath(BinDeps.depsdir(mpi), "src")
 
+if isdir(builddir)
```

Comment:
An alternative would be the Pythonish implementation
```Julia
try
    rm(prefix, recursive = true)
catch
    nothing
end
```
In a way this is safer since it tries to remove anything called `prefix`. On the other hand we lose all error handling.
Is this pattern common in Julia as well?